src/scenarioValidator.test.js

```diff
@@ -0,0 +1,59 @@
+/* globals expect */
+import scenarioValidator from './scenarioValidator';
+
+describe('Validate Scenario', () => {
+  it('Valid Scenario', () => {
+    const scenario = {
+      url: 'http://lol.co.uk/',
+      label: 'scenario-height-missing',
+      viewports: [{ width: 1024, height: 800, label: 'large' }]
+    };
+
+    expect(scenarioValidator(scenario));
+  });
+  it('Scenario missing height', () => {
+    const scenario = {
+      url: 'http://lol.co.uk/',
+      label: 'scenario-height-missing',
+      viewports: [{ width: 1024, label: 'large' }]
+    };
+    expect(() => scenarioValidator(scenario)).toThrow(
+      'scenario-height-missing has no height set'
```

Comment:
how does this work if there are multiple viewports defined?

It might be good to use the viewport label in the error